### PR TITLE
Add go.mod and go.sum to lock dependencies

### PR DESCRIPTION
**Pull request by Iceyer**

### CODE REVIEW

**Comment by Iceyer** [comment]
```log
$ docker pull vbatts/pandoc

Using default tag: latest

Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit

The command "docker pull vbatts/pandoc" failed and exited with 1 during .
```

could anyone help restart the ci build job?

@brendandburns @cyphar @jbouzane

**Comment by fuweid** [comment]
@Iceyer https://github.com/opencontainers/image-spec/pull/814 will fix the issue. After the pr merged, you can rebase/repush to restart CI.

**Comment by fuweid** [comment]
the CI issue has been fixed. Needs rebase here

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module github.com/opencontainers/image-spec
```
Comment (vbatts) [inline go.mod:1]: Do we want this at the top level? or just for specs-go?

Comment (vbatts) [reply go.mod:1]: I guess for the `.tools/` we'll want it at the top...

**Review by vbatts — APPROVED** [review]
LGTM

**Comment by vbatts** [comment]
didn't even realize this is a duplicate of #810, which was opened before this one.

**Pull request closed without merging**